docs/talks/scipy2019/submission.rst

```diff
@@ -0,0 +1,93 @@
+Zarr - scalable storage of tensor data for use in parallel and distributed computations
+=======================================================================================
+
+SciPy 2019 submission.
+
+
+Short summary
+-------------
+
+(target: 100 words)
+
+@@TODO
+
+
+Abstract
+--------
+
+(target: 500 words)
+
+Background
+~~~~~~~~~~
+
+Across a broad range of scientific disciplines, data are naturally
+represented and stored as N-dimensional typed arrays, also known as
+tensors. The volume of data being generated is outstripping our
+ability to analyse it, and scientific communities are looking for ways
+to leverage modern multi-core CPUs and distributed computing
+platforms, including cloud computing. Retrieval and storage of data is
+often the major bottleneck, and new approaches to data storage are
+needed to accelerate distributed computations and enable them to scale
+on a variety of platforms.
+
+Methods
+~~~~~~~
+
+We have designed a new storage format and protocol for tensor data
+[1_], and have released an open source Python implementation [2_,
+3_]. Our approach builds on data storage concepts from HDF5 [4_],
+particularly chunking and compression, and hierarchical organisation
+of datasets. Key design goals include: platform-independent and as
+simple as possible to implement in other programming languages;
+support for multiple concurrent readers or writers; support for a
+variety of parallel computing environments, from multi-threaded
+execution on a single CPU to multi-process execution across a
+multi-node cluster; pluggable storage subsystem with support for cloud
+object stores; pluggable encoding subsystem with support for a variety
```

Comment:
suggest: pluggable storage subsystem with support for cloud object stores, file systems, or key-value databases

Reply:
addressed